e2e/browser/src/components/appContainer/index.tsx

```diff
@@ -0,0 +1,88 @@
+import React, { useState, useEffect } from "react";
+import {
+  login,
+  logout,
+  handleIncomingRedirect,
+  getDefaultSession,
+} from "@inrupt/solid-client-authn-browser";
+import Notifications from "../notifications/";
+
+const REDIRECT_URL = "http://localhost:3000/";
+const APP_NAME = "Notifications browser-based tests app";
+const issuer = "https://login.inrupt.com/";
+
+const NotificationContainer = () => {
+  if (getDefaultSession().info.isLoggedIn) {
+    return <Notifications />;
+  } else {
+    return <div></div>;
+  }
+};
+
+export default function AppContainer() {
+  const [loggedIn, setLoggedIn] = useState<boolean>(false);
+  const [issuer, setIssuer] = useState<string>("https://login.inrupt.com");
```

Comment:
should reuse `issuer` from line 12, but rename that variable `DEFAULT_ISSUER`

Reply:
aa86165ebd61620ee42d0c6cdf9a27538606b1fe